Formula/jiri.rb

```diff
@@ -0,0 +1,35 @@
+class Jiri < Formula
+  desc "Tool to simplify multi-repo development"
+  homepage "https://fuchsia.googlesource.com/jiri"
+  url "https://fuchsia.googlesource.com/jiri.git",
```

Comment:
I'm sorry but this violates the acceptable formula policy https://docs.brew.sh/Acceptable-Formulae#stable-versions since it doesn't have a tagged version. Do you know if this is something the project is working towards?